netplan_cli/cli/state.py

```diff
@@ -39,16 +40,19 @@
 DEVICE_TYPES = {
     'bond': 'bond',
     'bridge': 'bridge',
+    'dummy': 'dummy',
     'ether': 'ethernet',
     'ipgre': 'tunnel',
     'ip6gre': 'tunnel',
     'loopback': 'ethernet',
     'sit': 'tunnel',
     'tunnel': 'tunnel',
+    'tun': 'tunnel',
     'tunnel6': 'tunnel',
     'wireguard': 'tunnel',
     'wlan': 'wifi',
     'wwan': 'modem',
+    'veth': 'veth',
```

Comment:
nitpick: Should this be translated to `virtual-ethernet` to stay in line with Netplan YAML?